src/components/DownloadLoginFile.js

```diff
@@ -119,16 +119,18 @@ class DownloadLoginFile extends Nimiq.Observable {
      * @param {MouseEvent} event
      */
     _onMouseDown(event) {
+        /** @type {HTMLElement} */
+        const target = (event.target);
+        if (target.matches('.continue')) {
+            this._onDownloadEnd();
+            return;
+        }
+
         if (event.button === 0) { // primary button
             if (!this._supportsNativeDownload()) return;
             this._onDownloadStart();
-        // } else if (event.button === 2) { // secondary button
-        //     // Big Problem with the secondary button listening:
-        //     // When user does right-click, but then does NOT save-as,
-        //     // then unfocues the window for whatever reason and then
-        //     // refocuses it, the download is considered finished
-        //     // and the request is returned.
-        //     this._onDownloadStart(true);
+        } else if (event.button === 2) { // secondary button
+            this._onDownloadStart(true);
```

Comment:
Might be a little bit simplified:
```js
if (!this._supportsNativeDownload()) return;
this._onDownloadStart(event.button === 2);
```

Reply:
This actually leads to a different result, has in the `_onDownloadStart` function is not called, when native download is not supported. In my existing code, the download-start function is still called for secondary-clicks, even if native download is not supported.